web_toggle/static/src/js/bs_toggle.js

```diff
@@ -0,0 +1,74 @@
+openerp.web_toggle = function(instance, local) {
+    var _t = instance.web._t,
+        _lt = instance.web._lt;
+    var QWeb = instance.web.qweb;
+
+local.FieldBooleanToggle = instance.web.form.AbstractField.extend({
+    template: 'FieldBooleanToggle',
+    start: function() {
+        var self = this;
+        this.$checkbox = $("input", this.$el);
```

Comment:
I didn't know that syntax ``$("input", this.$el)``, I usually use ``this.$el.find("input")``. I won't complain about I've tested it both works!